tests/gpu-tests/test_sandbox_mounts.py

```
@@ -0,0 +1,83 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from nemo_skills.pipeline.cli import run_cmd, wrap_arguments
+from tests.conftest import docker_rm
+
+
+@pytest.mark.gpu
+def test_sandbox_mounts_read_only_functional():
+    base_dir = Path("/tmp/nemo-skills-tests/sandbox-mounts-read-only")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/nemo-skills-tests/sandbox-mounts-read-only"

(S108)